native/core/src/execution/datafusion/expressions/sum_decimal.rs

```diff
@@ -112,6 +112,13 @@ impl AggregateExpr for SumDecimal {
             self.scale,
         )))
     }
+
+    fn default_value(&self, _data_type: &DataType) -> DFResult<ScalarValue> {
```

Comment:
```suggestion
    fn default_value(&self, _data_type: &DataType) -> Result<ScalarValue> {
```

Reply:
This is DataFusion API. It returns DataFusion's predefined `Result`.

Reply:
For other similar methods in this PR the import DataFusion Result is used instead of DFResult?

Reply:
correct, that was my suggestion to use the same alias across expression files, it can Result or DFResult, whatever is easier

Reply:
Okay. It is not related to this PR. We can change to use a consistent one in a separate PR.